README.md

```diff
@@ -196,14 +205,10 @@ to check the number of rows in table `mytable`.
 
 The tables created by HarbourBridge provide a starting point for evaluation of
 Spanner. While they preserve much of the core structure of your PostgreSQL/MySQL
-schema and data, many key features have been dropped, including (non-primary)
-indexes, functions, sequences, procedures, triggers, and views.
+schema and data, many key features have been dropped, including functions, sequences, procedures, triggers, and views. For DynamoDB, the conversion from schemaless to schema is focused on the use-case where customers use DynamoDB in a consistent, structured way with a fairly well defined set of columns and types.
 
 As a result, the out-of-the-box performance you get from these tables could be
-slower than what you get from PostgreSQL/MySQL. HarbourBridge does preserve primary
-keys, but we do not currently translate other indexes. If your SQL query
```

Comment:
Removed this since we convert indexes now.

Reply:
We should also edit the paragraph below where we say "To improve performance, consider adding [Secondary Indexes]" since HB already does that for the user.